store/storetest/notify_admin_store.go

```diff
@@ -145,6 +146,28 @@ func testGetDataByUserIdAndFeature(t *testing.T, ss store.Store) {
 	tearDown(t, ss)
 }
 
+func testUpdate(t *testing.T, ss store.Store) {
+	userId1 := model.NewId()
+	d1 := &model.NotifyAdminData{
+		UserId:          userId1,
+		RequiredPlan:    model.JenkinsPlugin,
+		RequiredFeature: model.WorkTemplates,
```

Comment:
For existing notify admin, RequiredPlan is the category, and RequiredFeature is the subcategory. I'd recommend doing the same for work templates for more understandable code, e.g. RequiredPlan is always `model.WorkTemplates`, and `RequiredFeature` is the plugin id